hugegraph-llm/src/hugegraph_llm/operators/index_op/build_semantic_index.py

```diff
@@ -39,9 +37,7 @@ def _extract_names(self, vertices: list[str]) -> list[str]:
 
     # TODO: use asyncio for IO tasks
     def _get_embeddings_parallel(self, vids: list[str]) -> list[Any]:
```

Comment:
should inline the method if we don't need the parallelization  (Also delete the comment)